pocs/observatory.py

```diff
@@ -507,6 +511,8 @@ def autofocus_cameras(self, camera_list=None, coarse=False):
                 else:
                     autofocus_events[cam_name] = autofocus_event
 
+        self._coarse_focus_done = True
```

Comment:
Along with above, seems like this should check if a coarse focus was done and if so set variable. It is currently always set to True as far as I can tell.

Reply:
It is all a bit odd. It sets `self._coarse_focus_done` to `True` after any call to `autofocus_cameras`, regardless of whether a coarse focus was actually performed.

I wondering whether we should just remove all of this. We hardly every want to actually do a coarse focus, and I feel like any logic for deciding whether to do it or not belongs more in the state machine than here.

Reply:
That's okay with me so I leave the decision up to you. We could feasibly build a coarse-focus state that we could transition to at various points.

Reply:
OK. Will remove it for now and see how we go during the next Huntsman run, and reinstate if needed.